java.io.FileNotFoundException: armeabi/obfs4proxy.mp3 and armeabi/pdnsd.mp3

Problem with master branch:
Not enough files: obfs4proxy.mp3 and pdnsd.mp3 after build in assets forlder.
Crash after start application:
> error onBind
> 	   java.io.FileNotFoundException: armeabi/obfs4proxy.mp3
> 		   at android.content.res.AssetManager.openAsset(Native Method)
> 		   at android.content.res.AssetManager.open(AssetManager.java:363)
> 		   at android.content.res.AssetManager.open(AssetManager.java:337)
> 		   at org.torproject.android.service.util.OtherResourceInstaller.installResources(OtherResourceInstaller.java:77)
> 		   at org.torproject.android.service.TorService.torUpgradeAndConfig(TorService.java:650)
> 		   at org.torproject.android.service.TorService.access$800(TorService.java:90)
> 		   at org.torproject.android.service.TorService$2.run(TorService.java:599)
> 		   at java.lang.Thread.run(Thread.java:818)

Same issue :(( @Tin-moon  Did you fix it?
02-09 11:34:17.449 8881-8985/org.torproject.android E/Orbot: error onBind
java.io.FileNotFoundException: armeabi/obfs4proxy.mp3
at android.content.res.AssetManager.openAsset(Native Method)
at android.content.res.AssetManager.open(AssetManager.java:318)
at android.content.res.AssetManager.open(AssetManager.java:292)
at org.torproject.android.service.util.OtherResourceInstaller.installResources(OtherResourceInstaller.java:77)
at org.torproject.android.service.TorService.torUpgradeAndConfig(TorService.java:650)
at org.torproject.android.service.TorService.access$800(TorService.java:90)
at org.torproject.android.service.TorService$2.run(TorService.java:599)
at java.lang.Thread.run(Thread.java:818)

And 
2-09 11:31:45.934 2450-7273/org.torproject.android1 D/OrbotVpnService: tun2Socks has stopped
java.io.IOException: Error running exec(). Command: -c, /data/data/org.torproject.android1/app_bin/pdnsd.conf Working Directory: null Environment: [ANDROID_ROOT=/system, EMULATED_STORAGE_SOURCE=/mnt/shell/emulated, LOOP_MOUNTPOINT=/mnt/obb, LD_PRELOAD=libsigchain.so:libNimsWrap.so, ANDROID_BOOTLOGO=1, EMULATED_STORAGE_TARGET=/storage/emulated, EXTERNAL_STORAGE=/storage/emulated/legacy, SYSTEMSERVERCLASSPATH=/system/framework/services.jar:/system/framework/ethernet-service.jar:/system/framework/wifi-service.jar, ANDROID_SOCKET_zygote=9, PATH=/sbin:/vendor/bin:/system/sbin:/system/bin:/system/xbin, ANDROID_DATA=/data, ANDROID_ASSETS=/system/app, ASEC_MOUNTPOINT=/mnt/asec, BOOTCLASSPATH=/system/framework/core-libart.jar:/system/framework/conscrypt.jar:/system/framework/okhttp.jar:/system/framework/core-junit.jar:/system/framework/bouncycastle.jar:/system/framework/ext.jar:/system/framework/framework.jar:/system/framework/telephony-common.jar:/system/framework/voip-common.jar:/system/framework/ims-common.jar:/system/framework/mms-common.jar:/system/framework/android.policy.jar:/system/framework/apache-xml.jar:/system/framework/qcmediaplayer.jar:/system/framework/org.codeaurora.Performance.jar:/system/framework/vcard.jar:/system/framework/tcmiface.jar:/system/framework/qcom.fmradio.jar:/system/framework/oem-services.jar, ANDROID_PROPERTY_WORKSPACE=8,0, SECONDARY_STORAGE=/storage/sdcard1, ANDROID_STORAGE=/storage]
at java.lang.ProcessManager.exec(ProcessManager.java:211)
at java.lang.ProcessBuilder.start(ProcessBuilder.java:195)
at org.torproject.android.service.vpn.OrbotVpnManager.startDNS(OrbotVpnManager.java:413)
at org.torproject.android.service.vpn.OrbotVpnManager.access$400(OrbotVpnManager.java:55)
at org.torproject.android.service.vpn.OrbotVpnManager$2.run(OrbotVpnManager.java:295)
Caused by: java.io.IOException: No such file or directory
at java.lang.ProcessManager.exec(Native Method)
at java.lang.ProcessManager.exec(ProcessManager.java:209)
at java.lang.ProcessBuilder.start(ProcessBuilder.java:195)
at org.torproject.android.service.vpn.OrbotVpnManager.startDNS(OrbotVpnManager.java:413)
at org.torproject.android.service.vpn.OrbotVpnManager.access$400(OrbotVpnManager.java:55)
at org.torproject.android.service.vpn.OrbotVpnManager$2.run(OrbotVpnManager.java:295)

--- a/orbotservice/src/main/jni/Android.mk
+++ b/orbotservice/src/main/jni/Android.mk
@@ -31,6 +31,13 @@ LOCAL_CFLAGS    := -Wall -O2 -I$(LOCAL_PATH)/pdnsd -DHAVE_STPCPY
 
 include $(BUILD_EXECUTABLE)
 
+.PHONY: $(CopyToAssets)
+
+CopyToAssets: libs/armeabi/pdnsd
+    $(shell mkdir -p assets/armeabi)
+    $(shell zip assets/armeabi/pdnsd.mp3 libs/armeabi/pdnsd)
+
+
 ########################################################
 ## libancillary
 ########################################################

